tutorials/introductory/usage.py

```diff
@@ -130,22 +130,23 @@
 #
 # Coding styles
 # =============
-
-# The object-oriented and the pyplot interfaces
-# ---------------------------------------------
+#
+# The explicit and the implicit approach of programming
+# -----------------------------------------------------
 #
 # As noted above, there are essentially two ways to use Matplotlib:
 #
-# - Explicitly create figures and axes, and call methods on them (the
-#   "object-oriented (OO) style").
-# - Rely on pyplot to automatically create and manage the figures and axes, and
-#   use pyplot functions for plotting.
+# - Explicitly create Figures and Axes, and call methods on them (the explicit
+#   or "object oriented programming (OOP) style").
```

Comment:
I'm not sure about "OOP" here.

Reply:
"OOP" appears to be one of the "correct" versions according to https://matplotlib.org/stable/devel/style_guide.html#terminology (last but one row in the table)

Reply:
Hmm, given that OOP is never used in the docs, nor the phrase "object-oriented programming", I'm not sure where this directive came from...  However, you are correct that that is what the style guide _says_....

Reply:
So shall I keep the changes or do you want to wait for the outcome of #21817? (I didn't know that a word as typically German as "verboten" had made it into English ☺️)

Reply:
I'm happy with most of the rest of this PR, but suggest you remove these changes.  I appreciate you followed the style guide here, and we should have a discussion of what the proper terms to use are, but the _current_ common usage is as written, _not_ what the style guide suggests.  If we decide to stick with what is in the style guide, then we can change here at the same time as the rest of the docs.

Reply:
reverted the OO/pyplot/explicit/implicit changes